### ISSUE TITLE
Better user-friendly effect options names

### ISSUE THREAD

**Issue by mwestphal** [f3d-app/f3d#344]
```
  this->Internals->init("render.effect.depth-peeling", false); // MEAK: "depth-peeling" -> "translucency-support"?
  this->Internals->init("render.effect.fxaa", false); // MEAK: "fxaa" -> "anti-aliasing"?
  this->Internals->init("render.effect.ssao", false); // MEAK: "ssao" -> "ambient-occlusion"?
  this->Internals->init("render.effect.tone-mapping", false);
```

**Comment by mwestphal** [comment]
wdyt @Meakk ?

**Comment by Meakk** [comment]
I agree, let's brainstorm again on it before the release.

**Comment by mwestphal** [comment]
Fixed by #688
